deploy/manifests/dev/us-east-2/tenant/storetheindex/dhstore/deployment.yaml

```diff
@@ -19,10 +19,10 @@ spec:
           resources:
             limits:
               cpu: "3"
-              memory: 10Gi
+              memory: 20Gi
             requests:
               cpu: "3"
-              memory: 10Gi
+              memory: 20Gi
```

Comment:
As is the chances are this manifest is going to get scheduled on expensive node types or not get scheduled at all.

We have created a memory optimised worker group to run `ago` before. 

I recommend configuring node affinity for this deployment to use that.

Check that the AZ for the node group matches the AZ of PVC used by dhstore. If not create a new worker node group with the same instance type as the ones used by `ago` and use that.